rpc_jobs/periodic_cleanup.yml

```diff
@@ -117,4 +117,23 @@
             throw e
           }
         } // internal_slave
+        stage("Master Log Compression"){
+          node("master"){
+            // Note that already compressed logs will be skipped as they
+            // get renamed .gz .
+            // This only affects the stage logs used for blue ocean,
+            // console logs are compressed using a plugin activated via
+            // common.globalWraps.
+            // Logs newer than 1 day are skipped to prevent compressing
+            // the logs of jobs that are currently running.
+            sh """#!/bin/bash
+              cd /var/lib/jenkins/jobs
+              find . -name \\*.log -mtime +1 \\
```

Comment:
It looks like there are other things, e.g. workspaces, in the job directories in addition to log files. While unlikely, it seem like there is the potential for this to modify a live build. What are your thoughts on something a bit more targetted, e.g. `find . -regex '.+/builds/[0-9]+/[0-9]+\.log' -mtime +1`